mslib/mscolab/conf.py

```diff
@@ -76,6 +76,12 @@ class default_mscolab_settings:
     # used to generate the password token
     SECURITY_PASSWORD_SALT = secrets.token_urlsafe(16)
 
+    # Max allowed file size for uploading user profile image
+    MAX_IMAGE_SIZE = 1 * 1024 * 1024  # 1MB
```

Comment:
we have also attachments which can be images in the chat manager. The var maybe later used there too.